client/src/pages/dashboard/teams/[teamId]/edit.jsx

```diff
@@ -91,6 +128,36 @@ const edit = () => {
                         {...register('teamName')}
                         isRequired
                     />
+                    <Stack spacing={3} overflow="auto" maxHeight="250px">
+                        <FormLabel fontSize="1.25rem">Current players</FormLabel>
+                        {players.map((player) => {
+                            return (
+                                <HStack key={player?.id} spacing="0.5rem" align="center">
+                                    <Editable
+                                        minW={isDesktop ? '320px' : '160px'}
+                                        size="sm"
+                                        defaultValue={player?.name}
+                                        isDisabled="true"
+                                    >
+                                        <EditablePreview />
+                                        <EditableInput />
+                                    </Editable>
+                                    <CloseButton
+                                        size="sm"
+                                        onClick={() => {
+                                            setSelected({
```

Comment:
Since this depends on the previous state, would we have to make this async through a callback?
```suggestion
                                            setSelected(prevSelected => { playerIds: prevSelected.playerIds.concat([player?.id]) })
```